upgrade/upgrade-scripts/upgrade-container

```diff
@@ -527,8 +527,23 @@ function do_upgrade_container_not_in_place() {
 		die "failed to migrate configuration for '$CONTAINER'"
 }
 
+function get_type() {
+	local root origin
+
+	root=$(sudo zfs list -Hpo name /var/lib/machines/${CONTAINER})
```

Comment:
remove `sudo` here; we already force the caller to be root, so `sudo` is unnecessary.